src/cleanup_rules/kt/rules.toml

```diff
@@ -488,7 +488,7 @@ name = "delete_file_with_no_declarations"
 query = """
 (source_file) @source_file"""
 replace = ""
-replace_node = "program"
```

Comment:
The changes  logic (below), now disallows wrong rules that reference non-existing variables